routes/personality_route.py

```
@@ -1,17 +1,13 @@
 # FastAPI
-from typing import List
-from fastapi import APIRouter, status, HTTPException
+from fastapi import APIRouter, status
 from fastapi.params import Query, Depends
 from fastapi.responses import JSONResponse
-from pydantic import BaseModel
-from sqlalchemy.orm import Session
 
-from crud.personality import get_latest_personality_by_user_id, create_personality
+from crud.chat_log import get_recent_user_messages
+from crud.personality import *
```

Ruff (0.8.2)

`from crud.personality import *` used; unable to detect undefined names

(F403)